src/components/Feed/components/AddComment.js

```diff
@@ -1,13 +1,9 @@
 import * as React from "react";
```

Comment:
Prob delete this file since not having adding comments

Reply:
Good idea